Simulator/Recorders/Hdf5Recorder.cpp

```diff
@@ -122,6 +123,94 @@ void Hdf5Recorder::saveSimData(const AllVertices &neurons)
    }
 }
 
+// Processes and updates HDF5 datasets for variables marked as DYNAMIC
+void Hdf5Recorder::compileHistories(AllVertices &vertices)
```

Comment:
Does this method still need this parameter?

Reply:
no, they don't need parameter any more